### ISSUE TITLE
DAC 13 - Medium Priority WCAG Level AA - Focus Visible

### ISSUE THREAD

**Issue by JadeneAd** [ONSvisual/income-scrolly#13]
<img width="314" alt="DAC_Focus_Visible_01" src="https://user-images.githubusercontent.com/75850106/155557467-34addfcf-1677-43ca-b4cc-8bb6b66effc0.png">

### Issue
Keyboard only users may not know whether their keyboard focus position has moved onto the ‘Animation off/on’ checkbox element. This is because there is no visible focus indicator.

<img width="599" alt="DAC 13 Current code ref" src="https://user-images.githubusercontent.com/75850106/155557687-8b0d94ad-50a1-4328-9220-9a925bf335bf.png">

**Keyboard only comments:**
“With the second tab, focus disappears. It appears that the ‘animation off’ toggle element does not have a visible focus. This was disorientating as I was unsure where my focus was.”

### Suggested Solution
Ensure that there is an obvious focus indicator and that the colour contrast used is 3:1 against adjacent colours.
